general/deploy.sh

```diff
@@ -34,14 +34,12 @@ if [ -d "./$SERVICE" ]; then
     echo "========================================"
     echo "SSH Preparation"
     
-    echo "$SSH_PRIVATE_KEY" >> ./id_rsa
-    echo "$SSH_PUBLIC_KEY" >> ./id_rsa.pub
-    chmod 700 id_rsa
-    chmod 700 id_rsa.pub
-    
-    mkdir -p ~/.ssh/
-    # TODO: IMPLEMENT THIS https://serverfault.com/questions/132970/can-i-automatically-add-a-new-host-to-known-hosts/807363#807363
-    ssh-keyscan -H $SERVER_HOST >> ~/.ssh/known_hosts
+    mkdir -p ~/.ssh && chmod 700 ~/.ssh
+    echo "$SSH_PRIVATE_KEY" >> ~/.ssh/id_rsa && echo "$SSH_PUBLIC_KEY" >> ~/.ssh/id_rsa.pub
+    chmod 600 ~/.ssh/id_rsa && chmod 644 ~/.ssh/id_rsa.pub
+    cat ./general/ssh-config-file > ~/.ssh/config && chmod 644 ~/.ssh/config
```

Comment:
Oh ya, satu lagi. Bikin jadi gini dong.

```suggestion
    cat ./general/ssh-config-file >> ~/.ssh/config && chmod 644 ~/.ssh/config
```

Lebih safe append line daripada overwrite deh kayanya, takut ada unexpected behaviour juga di github action nya